tests/regression/mmseg.yml

```diff
@@ -211,8 +211,8 @@ models:
     model_configs:
       - configs/apcnet/apcnet_r50-d8_512x1024_40k_cityscapes.py
     pipelines:
-      - *pipeline_ort_dynamic_fp32
-      - *pipeline_trt_dynamic_fp16
+      - *pipeline_ort_static_fp32
```

Comment:
what's the motivation of make changes to `mmseg.yml`?

Reply:
apcnet only supports static shape as described https://github.com/open-mmlab/mmdeploy/issues/1139